template/api.graphql

```diff
@@ -1,60 +1,91 @@
 ### This file was generated by Nexus Schema
 ### Do not make changes to this file directly
 
-
-"""Payload returned if login or signup is successful"""
+"""
+Payload returned if login or signup is successful
+"""
 type AuthPayload {
-  """The current JWT token. Use in Authentication header"""
+  """
+  The current JWT token. Use in Authentication header
+  """
   token: String
 
-  """The logged in user"""
+  """
+  The logged in user
+  """
   user: User
 }
 
 """
-A date-time string at UTC, such as 2007-12-03T10:15:30Z, compliant with the
-`date-time` format outlined in section 5.6 of the RFC 3339 profile of the ISO
-8601 standard for representation of dates and times using the Gregorian calendar.
+A date-time string at UTC, such as 2007-12-03T10:15:30Z, compliant with the `date-time` format outlined in section 5.6 of the RFC 3339 profile of the ISO 8601 standard for representation of dates and times using the Gregorian calendar.
 """
 scalar DateTime
 
 input DateTimeFilter {
   equals: DateTime
   gt: DateTime
   gte: DateTime
-  in: [DateTime!]
+  in: [DateTime]
   lt: DateTime
   lte: DateTime
-  not: DateTime
-  notIn: [DateTime!]
+  not: NestedDateTimeFilter
+  notIn: [DateTime]
 }
 
 """
-The `JSON` scalar type represents JSON objects as specified by [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf).
+The `JSONObject` scalar type represents JSON objects as specified by [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf).
 """
-scalar Json
+scalar JSONObject
 
 type Mutation {
   createUser(data: UserCreateInput!): User!
 
-  """Login to an existing account"""
+  """
+  Login to an existing account
+  """
   login(email: String!, password: String!): AuthPayload
 
-  """Signup for an account"""
+  """
+  Signup for an account
+  """
   signup(data: SignupInput!): AuthPayload
 }
 
-enum OrderByArg {
-  asc
-  desc
+input NestedDateTimeFilter {
+  equals: DateTime
+  gt: DateTime
+  gte: DateTime
+  in: [DateTime]
+  lt: DateTime
+  lte: DateTime
+  not: NestedDateTimeFilter
+  notIn: [DateTime]
+}
+
+input NestedStringFilter {
```

Comment:
Curious to why this exists being that there doesn't appear to be any difference from `StringFilter` as far as I can tell?
Maybe just semantics in the name?

Same thought on the other types with the `Nested{Type}` 🤔

Reply:
Good question. It looks like we forgot to re-run this during an upgrade. After all the open PRs are merged, I'll re-run and check the diff.